test/features/bootstrap_init_join_remove.feature

```diff
@@ -0,0 +1,26 @@
+@bootstrap
+Feature: HA bootstrap process - init, join and remove
+
+  Scenario: Init cluster service on node "hanode1", and join on node "hanode2"
+    Given   Cluster service is "stopped" on "local"
```

Comment:
Maybe we should use `hanode1` instead of `local` ?

Reply:
changed